docs/content/Caching/Using-Pre-Aggregations.md

```diff
@@ -485,3 +356,78 @@ services:
 [ref-config-extdriverfactory]: /config#options-reference-external-driver-factory
 [ref-production-checklist-refresh]:
   /deployment/production-checklist#set-up-refresh-worker
+
+## Pre-Aggregations Storage
+
+The default pre-aggregations storage in Cube.js is its own purpose-built storage layer: Cube Store.
+
+Alternatively, you can store pre-aggregations either **internally** in the source database, or **externally**  in databases such as MySQL or Postgres.
```

Comment:
I believe it totally makes sense to add link to this blog post or reuse parts of its content (the schemas at the very least) here: https://cube.dev/blog/introducing-cubestore/

Reply:
+1 to @igorlukanin's suggestion, but it might be one for a separate PR. This PR was specifically to merge the known limitations sections; we can create a new one to track improving this section IMO.